api/api.py

```diff
@@ -669,3 +669,17 @@ def docker_prune_settings(disable: bool=True, max_age_mins: int=360, num_builds:
     interval_hrs: run a Docker Prune every ``interval_hrs`` hours (unless ``num_builds`` is set, in which case use the "prune every X builds" logic). Defaults to 1 hour
   """
   pass
+
+def analytics_settings(enable: bool) -> None:
```

Comment:
Think it's worth making a cloud module, to point where we're going?

Reply:
i don't understand what you're asking

Reply:
The same way we have an os module in the Tiltfile, have a module called cloud, that would configure the settings for how local Tilt interacts with Tilt Cloud.

Reply:
hmmm...i don't see these as being related. the current api is for consistency with existing tilt commands, e.g., 'tilt analytics' -> 'analytics_settings', 'tilt docker-prune', -> 'docker_prune_settings', etc